esp/bindings/http/platform/httptransport.cpp

```diff
@@ -798,6 +798,44 @@ void CHttpMessage::logSOAPMessage(const char* message, const char* prefix)
     return;
 }
 
+static bool skipLogContent(const char* httpHeader)
+{
+    VStringBuffer postStr("%s ", POST_METHOD);
+    if (!startsWith(httpHeader, postStr.str()))
+        return false;
+
+    const char* servicePtr = httpHeader + 5;
+    if (isEmptyString(servicePtr) || (servicePtr[0] != '/'))
+        return false;
+
+    const char* methodPtr = strchr(++servicePtr, '/');
+    if (!methodPtr)
+        return false;
+
+    StringBuffer espService;
+    espService.append(methodPtr - servicePtr, servicePtr);
+
+    StringBuffer espMethod;
+    const char* tail = strchr(++methodPtr, '.');
+    if (tail && (startsWithIgnoreCase(tail, ".xml") || startsWithIgnoreCase(tail, ".json")))
+        espMethod.append(tail - methodPtr, methodPtr);
+    else
+    {
+        tail = strchr(methodPtr, '?');
+        if (!tail)
+            tail = strchr(methodPtr, ' ');
+        if (tail)
+            espMethod.append(tail - methodPtr, methodPtr);
+        else
+            espMethod.append(methodPtr);
+    }
+
+    if (strieq(espService, "ws_access"))
```

Comment:
If changed to check service above, then we can just check method here.
Either just check for "AddUser", "UserResetPass", or "UpdateUser".

Or if you want to be careful that UpdateUser is only for ws_account, etc, then set a flag above when you match the service name so you know which one it is.